src/main/App.ts

```diff
@@ -26,20 +26,29 @@ import { Schedule } from './entity/Schedule';
 import { SavedPromotion } from './entity/SavedPromotion';
 import redis, { RedisClient } from 'redis';
 import { CachingService } from './service/CachingService';
+import { initFirebaseAdmin } from './service/FirebaseConfig';
```

Comment:
I don't think `FirebaseConfig` should be in the service directory. I think usually any classes inside the service directories are classes we want to inject through constructor and perform some logic like `CachingService`. Could we move `FirebaseConfig` one level up? It will be in same directory as `App.ts` and `Index.ts`

Reply:
Sure!